api/src/main/kotlin/com/gmtkgamejam/Application.kt

```diff
@@ -22,13 +21,14 @@ fun Application.module() {
         modules(DatabaseModule, DiscordBotModule)
     }
 
-    configureRequestHandling()
-    configureUserInfoRouting()
-    configureAuthRouting()
     configureAdminRouting()
-    configurePostRouting()
-    configureFavouritesRouting()
+    configureAuthRouting()
     configureDiscordBotRouting()
+    configureFavouritesRouting()
+    configureInfraRouting()
+    configurePostRouting()
+    configureRequestHandling()
+    configureUserInfoRouting()
```

Comment:
The `infra` routing is the only new thing here, I've just sorted the list to be alphabetically ordered.